cluster/ubuntu-cluster/configure.sh

```diff
@@ -21,6 +21,10 @@
 
 set -e
 
+CONFIG_DIR=`dirname "$0"`
+CONFIG_DIR=`cd "$CONFIG_DIR"; pwd`
```

Comment:
Does this line really needed? Although `dirname` returns `./ubuntu-cluster`, but `cd CONFIG_DIR` can also works as you expected.